common/web/gesture-recognizer/src/engine/gestureRecognizer.ts

```diff
@@ -12,39 +10,9 @@ export class GestureRecognizer<HoveredItemType> extends TouchpointCoordinator<Ho
   private readonly mouseEngine: MouseEventEngine<HoveredItemType>;
   private readonly touchEngine: TouchEventEngine<HoveredItemType>;
 
-  protected static preprocessConfig<HoveredItemType>(
```

Comment:
And ... now I see that my comments relate to pre-existing code. I still think they are good comments though :grin: